src/Sarif/Errors.cs

```diff
@@ -509,6 +510,28 @@ public static RuntimeConditions LogUnhandledEngineException(IAnalysisContext con
             return RuntimeConditions.ExceptionInEngine;
         }
 
+        public static void LogIncompatibleRule(IAnalysisContext context, string ruleId, string incompatibleRuleId)
```

Comment:
>LogIncompatibleRule [](http://example.com/codeflow?start=27&length=19)

In general, when you update a user-facing message, please provide an example of the message from a real world test, so that we can assess its quality and completeness.